cms/templates/videos_index.html

```diff
@@ -61,13 +61,15 @@ <h3 class="sr">${_("Page Actions")}</h3>
             <div class="bit">
                 <h3 class="title-3">${_("Why upload video files?")}</h3>
                 <p>${_("For a video to play on different devices, it needs to be available in multiple formats. After you upload an original video file in .mp4 or .mov format on this page, an automated process creates those additional formats and stores them for you.")}</p>
+                <h3 class="title-3">${_("Maximum Video File Size")}</h3>
+                <p>${_("The maximum size of video file you can upload is 5 GB. If you try to upload a file larger than 5 GB, the upload will fail.")}</p>
                 <h3 class="title-3">${_("Monitoring files as they upload")}</h3>
                 <p>${_("Each video file that you upload needs to reach the video processing servers successfully before additional work can begin. You can monitor the progress of files as they upload, and try again if the upload fails.")}</p>
                 <h3 class="title-3">${_("Managing uploaded files")}</h3>
```

Comment:
The wording in this section was chosen to reflect the time lag between the edX end of the process and the YouTube end of the process. I defer to @rgrubb however.